Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -196,6 +196,15 @@ - (void)alertView:(UIAlertView *)alertView didDismissWithButtonIndex:(NSInteger)
                     // Only attach data once all log messages have been retrieved.
                     if (logStoresToLogMessagesMap.count == logStores.count) {
                         NSMutableString *emailBody = [NSMutableString stringWithFormat:@"%@\n", self.prefilledEmailBody];
+                        
+                        NSDictionary *emailBodyAdditions = [self.emailBodyAdditionsDelegate emailBodyAdditionsForEmailBugReporter:self];
```

Comment:
The email bug reporter isn't set up with proper DI, so getting a unit test here would get pretty messy. This has been manually tested, and I don't expect this file to undergo much churn.